src/main/kotlin/com/intellij/ml/llm/template/utils/Utils.kt

```diff
@@ -0,0 +1,33 @@
+package com.intellij.ml.llm.template.utils
+
+import com.intellij.ml.llm.template.extractfunction.EFSuggestion
+import com.intellij.ml.llm.template.extractfunction.EFSuggestionList
+
+fun addLineNumbersToCodeSnippet(codeSnippet: String, startIndex: Int): String {
+    val lines = codeSnippet.lines()
+    val numberedLines = lines.mapIndexed { index, line -> "${startIndex + index}. $line" }
+    return numberedLines.joinToString("\n")
+}
+
+
+fun replaceGithubUrlLineRange(githubUrl: String, lineStart: Int, lineEnd: Int): String {
+    val newLineRange = String.format("L$lineStart-L$lineEnd")
+    return String.format("${githubUrl.substringBefore("#")}#$newLineRange")
+}
+
+fun identifyExtractFunctionSuggestions(input: String): EFSuggestionList {
+    val regex = """\{"function_name":\s*"([^"]+)",\s*"line_start":\s*(\d+),\s*"line_end":\s*(\d+)\}""".toRegex()
```

Comment:
Could you please add a comment describing what we are looking for here? Maybe we can make `EFSuggestionList` serializable and parse it using Serializer?

Reply:
BTW, do we have a test for this regex?